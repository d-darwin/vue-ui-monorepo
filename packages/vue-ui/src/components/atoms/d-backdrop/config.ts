import { CssPosition } from "./types";
import styles from "./index.css?module";

export default {
  name: "DBackdrop",
  class: styles.dBackdrop,
  opacity: 0.5,
  zIndex: 1000,
  position: CssPosition.fixed,
} as const;
