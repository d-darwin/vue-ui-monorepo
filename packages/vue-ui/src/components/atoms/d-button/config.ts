export default {
  name: "DButton",
  className: "dButton",
};
