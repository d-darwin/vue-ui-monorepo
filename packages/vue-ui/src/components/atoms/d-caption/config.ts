import styles from "./index.css?module";

export default {
  name: "DCaption",
  class: styles["dCaption"],
} as const;
