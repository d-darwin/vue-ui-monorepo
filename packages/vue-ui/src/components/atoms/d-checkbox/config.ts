export default {
  name: "DCheckbox",
  className: "dCheckbox",
  inputClassName: "input",
  labelClassName: "label",
  labelInnerClassName: "labelInner",
  errorClassName: "error",
  iconContainerClassName: "iconContainer",
  iconContainerBackdropClassName: "iconContainerBackdrop",
  iconClassName: "icon",
  checkMark: "âœ“",
};
