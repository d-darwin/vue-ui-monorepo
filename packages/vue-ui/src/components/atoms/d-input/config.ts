export default {
  name: "DInput",
  className: "dInput",
};
