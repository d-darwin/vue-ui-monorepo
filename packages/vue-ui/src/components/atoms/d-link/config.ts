export default {
  name: "DLink",
  className: "dLink",
};
