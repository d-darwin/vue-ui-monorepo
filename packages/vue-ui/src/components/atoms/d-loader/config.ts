export default {
  name: "DLoader",
  className: "dLoader",
  wrapperClassName: "wrapper",
  backdropClassName: "backdrop",
  key: "loader",
  backdropKey: "backdrop",
  defaultAnimationDuration: "650ms",
  defaultZIndex: 1001,
  defaultContent: "â˜¯",
} as const;
