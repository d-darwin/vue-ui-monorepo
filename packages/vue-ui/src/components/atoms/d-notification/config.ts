export default {
  name: "DNotification",
  className: "dNotification",
} as const;
