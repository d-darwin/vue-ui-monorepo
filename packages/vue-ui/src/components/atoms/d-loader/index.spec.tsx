import { shallowMount } from "@vue/test-utils";
import DLoader from "@/components/atoms/d-loader";
import config from "./config";

describe("DLoader", () => {
  const wrapper = shallowMount(DLoader);

  it("Should render .dLoader element", async () => {
    const loaderEl = wrapper.find(`.${config.className}`);
    expect(loaderEl.exists()).toBeTruthy();
  });
});
