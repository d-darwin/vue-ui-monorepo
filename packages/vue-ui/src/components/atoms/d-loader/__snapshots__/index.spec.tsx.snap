// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DLoader Should render props.content 1`] = `
<div class="wrapper">
  <!---->
  <transition-stub enteractiveclass="transitionEnterActive" leaveactiveclass="transitionLeaveActive" appear="true" persisted="false" css="true">
    <div class="dLoader colorSchemePrimary fontHuge roundingFull sizeMedium transitionFast" style="--z-index: 1001; --animation-duration: 650ms;">
      <div>...</div>
    </div>
  </transition-stub>
</div>
`;
